### ISSUE TITLE
KeyError: 'image_type' when using `find_latest_image`

### ISSUE THREAD

**Issue by TMiguelT** [BioContainers/bioconda2biocontainer#1]
```python
from bioconda2biocontainer.biocontainer import find_package_by_name, find_package_by_term, find_latest_image
image = find_latest_image('abacas', 'v1.3.1-5-deb', all=False, sort_by_size=False, sort_by_download=False, container_type="Docker", registry_host=None)
```
```
Traceback (most recent call last):
  File "/media/michael/Storage2/JetBrains/apps/PyCharm-P/ch-0/201.8743.11/plugins/python/helpers/pydev/pydevd.py", line 1438, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/media/michael/Storage2/JetBrains/apps/PyCharm-P/ch-0/201.8743.11/plugins/python/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/media/michael/Storage2/Programming/BiocondaCli/bioconda_cli/main.py", line 101, in <module>
    main()
  File "/media/michael/Storage2/Programming/BiocondaCli/bioconda_cli/main.py", line 18, in main
    func(**kwargs)
  File "/media/michael/Storage2/Programming/BiocondaCli/bioconda_cli/__init__.py", line 17, in list_images
    image = find_latest_image(package['name'], latest_version['meta_version'], all=False, sort_by_size=False, sort_by_download=False, container_type="Docker", registry_host=None)
  File "/media/michael/Storage2/Programming/BiocondaCli/condaenv/lib/python3.8/site-packages/bioconda2biocontainer/biocontainer.py", line 54, in find_latest_image
    if filter_by_container_registry(container_type, registry_host, i):
  File "/media/michael/Storage2/Programming/BiocondaCli/condaenv/lib/python3.8/site-packages/bioconda2biocontainer/biocontainer.py", line 36, in filter_by_container_registry
    container_type == i['image_type']:
KeyError: 'image_type'
```

**Comment by TMiguelT** [comment]
Oh, actually I get these types of errors even with other parameter values, e.g.
```python
from bioconda2biocontainer.biocontainer import find_package_by_name, find_package_by_term, find_latest_image
image = find_latest_image('abacat', '0.0.4a', all=False, sort_by_size=False, sort_by_download=False, container_type=None, registry_host=None)
```
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/media/michael/Storage2/Programming/BiocondaCli/condaenv/lib/python3.8/site-packages/bioconda2biocontainer/biocontainer.py", line 63, in find_latest_image
    versions = sorted(versions, key=lambda i: i['updated'], reverse=True)
  File "/media/michael/Storage2/Programming/BiocondaCli/condaenv/lib/python3.8/site-packages/bioconda2biocontainer/biocontainer.py", line 63, in <lambda>
    versions = sorted(versions, key=lambda i: i['updated'], reverse=True)
KeyError: 'updated'
```

**Comment by r78v10a07** [comment]
Thanks for reporting this bug.
I fixed it in the 0.0.4 version
